tests/07reshape5intr

```diff
@@ -31,7 +31,19 @@ do
   echo 1000 > /proc/sys/dev/raid/speed_limit_min
   echo 2000 > /proc/sys/dev/raid/speed_limit_max
   check wait
-  while ! echo check > /sys/block/md0/md/sync_action; do sleep 0.1; done
+
+  counter=0
+  max_count=50  # 50 iterations of 0.1s = 5 seconds
+ 
+  while ! echo check > /sys/block/md0/md/sync_action; do
```

Comment:
Please use for or rework because exit condition should not be checked inside while (code readability reasons)

```
for ((i = 0 ; i < max ; i++ )); do
    if [[ echo check > /sys/block/md0/md/sync_action ]]; then
        break;
done

if [[ i == max ]]; then
     echo >&2 "Timeout waiting for check to succeed"
     exit 1
fi
```